src/groupeddataframe/utils.jl

```diff
@@ -346,9 +346,13 @@ function row_group_slots(cols::NTuple{N, AbstractVector},
                 seeni = seen_vec[Threads.threadid()]
                 local refs_i
                 let i=i # Workaround for julia#15276
-                    refs_i = map(c -> @inbounds c[i], refarrays)
+                    refs_i = map(refarrays) do c
+                        return @inbounds c[i]
+                    end
+                end
+                vals = map(refmaps, refs_i, strides, firstinds) do (m, r, s, fi)
+                    return @inbounds m[r-fi+1] * s
                 end
-                vals = map((m, r, s, fi) -> @inbounds m[r-fi+1] * s, refmaps, refs_i, strides, firstinds)
```

Comment:
What's the point of this change?

Reply:
for some reason the following is not parsed correctly:
```
julia> map(x -> @inbounds x[1], [[1],[2]])
ERROR: MethodError: no method matching (::var"#7#8")()
```
so either a `do` block should be used (which I found clearer), or
```
map(x -> @inbounds(x[1]), [[1],[2]])
```
(with parens)